packages/application-components/src/components/page-not-found/page-not-found.tsx

```diff
@@ -16,7 +16,7 @@ const PageNotFound = () => (
       >
         {...messages.paragraph1}
         values={{
-          a: msg => (
+          a: (msg: React.ReactNodeArray) => (
```

Comment:
Yeah this also works but it's more of a workaround. Thanks

Reply:
Yeah I know.

Reply:
What his this attempting to tell me now?

<img width="898" alt="CleanShot 2019-08-12 at 12 32 51@2x" src="https://user-images.githubusercontent.com/1877073/62859409-562c4380-bcfd-11e9-88b6-f764014fe8a6.png">

Reply:
I think we should keep the version pinned to '3.1.1' until the issue I reported is fixed

Reply:
I downgraded react-intl to 3.1.1

Reply:
> What his this attempting to tell me now?

It's related to the bug I reported